index.html

```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
           </div>
         </div>
         </div> 
-      </section>
-    
+    </section>
+
+    <section class="contact-container">
+      <div class="frame42">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+        <input type="text" name="text" required placeholder="Text Input">
```

Comment:
- [ ] Please make sure the Name field accepts up to 30 chars, You can use the `maxLength` attribute.